src/system/ThemeModule/Controller/VarController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\ThemeModule\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; // used in annotations - do not remove
+use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\HttpFoundation\Request;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\Core\Theme\Annotation\Theme; // used in annotations - do not remove
+
+class VarController extends AbstractController
+{
+    /**
+     * Configure a theme's variables based on provided .yml definitions for each field.
+     * @Route("/admin/var/{themeName}")
+     * @todo change route name to /admin/variable/{themeName} when similar named is removed?
+     * @Theme("admin")
+     *
+     * @param Request $request
+     * @param string $themeName
+     * @return mixed
+     * @throws \InvalidArgumentException if theme type is not twig-based
+     */
+    public function varAction(Request $request, $themeName)
+    {
+        $themeBundle = \ThemeUtil::getTheme($themeName);
+        if (!$themeBundle->isTwigBased()) {
+            throw new \InvalidArgumentException('Theme type must be twig-based in ' . __FILE__ . ' at line ' . __LINE__ . '.');
+        }
+        $themeVarsPath = $themeBundle->getConfigPath() . '/variables.yml';
+        if (file_exists($themeVarsPath)) {
```

Comment:
Instead of wrapping all the stuff inside the if, rather invert the condition to decrease indenting.